openapi/openapi.go

```diff
@@ -74,17 +76,21 @@ func LoadFromYaml(openApiFilepath string) (*OpenAPI, error) {
 	return &api, err
 }
 
+// ResolveReferences traverses the entire openapi struct tree and loads te h
```

Comment:
`and load te h`?

Reply:
I mean, we know what it loads. I don't know why this comment is incomplete. Will fix in followup